ArduCopter/system.cpp

```diff
@@ -281,6 +281,11 @@ void Copter::startup_INS_ground()
     ahrs.reset();
 }
 
+// update the harmonic notch filter center frequency dynamically based on the throttle
+void Copter::update_dynamic_notch() {
```

Comment:
Bracket - sure.
Ummn, moving to the imu - I'm pretty sure that was where I had it originally but then was asked to move it :)
Seriously though, in the future we will hopefully have other ways of updating the frequency - e.g. FFT or DShot rather than throttle and I don't think that logic should live in the IMU code. I would prefer it here unless you feel strongly about it?